mudz.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 14
```

Comment:
Meh, feel free to undo the version stuff. Clearly I'm on VS2015 RC, will to change this back with notepad as my VS will constantly change it back again.